src/browser/process.rs

```diff
@@ -170,6 +179,14 @@ impl Process {
             args.extend(&["--headless"]);
         }
 
+        let extension_args: Vec<String> = launch_options
+            .extensions
+            .iter()
+            .map(|e| format!("--load-extension={}", e.display()))
```

Comment:
There is room for improvement here as Command [arguments](https://doc.rust-lang.org/std/process/struct.Command.html#method.arg) are `AsRef<OsStr>`, not `String`, to begin with. `PathBuf` already is `AsRef<OsStr>`. Going from `PathBuf` via `fmt::Display` to `String` and back to `OsStr` is unnecessary. The current `args`-type also allows to specify arguments which are actually invalid.